### DISCUSSION TITLE
2023년 개발자로서의 삶

### DISCUSSION THREAD

**Discussion by giscus** [kyY00n/devlog#6] [General]
# 2023년 개발자로서의 삶

https://kyy00n.vercel.app/posts/2

**Comment by parkmuhyeun** [comment]
2023년 고생많았고 오히려 제가 많이 배워서 감사하다고 전하고 싶네요~!

2024년도 잘해봅시다 🥕

**Comment by junpakPark** [comment]
윤로지와 함께여서 최고였던 우테코 5기~ :+1:

**Comment by euijinkk** [comment]
멋지십니다~!!
